index.html

```diff
@@ -272,134 +315,188 @@ <h2>Glossary</h2>
 
-<p><dfn class="lint-ignore export">Paragraph direction</dfn>. The initial <a>base direction</a>  of  a paragraph or string, which resolves to either <em>left-to-right</em> or <em>right-to-left</em>. Nested embedding controls may be used to change the direction of an inline range of text, but the paragraph direction sets the starting point which the <a>Unicode Bidirectional Algorithm</a> uses  to calculate the directions of the embedded levels. For more details, see <a href="http://www.unicode.org/reports/tr9/">Unicode Standard Annex #9, Unicode Bidirectional Algorithm</a> [[UAX9]], especially definitions BD2–BD5.<br><span class="see-also">See also the definition in the <a href="https://www.unicode.org/glossary/#paragraph_direction">Unicode Standard</a>.</span></p>
+<p><dfn class="lint-ignore export">Paragraph direction</dfn>. The initial <a>base direction</a> of  a paragraph or string, which resolves to either <em>left-to-right</em> or <em>right-to-left</em>. Nested embedding controls may be used to change the direction of an inline range of text, but the paragraph direction sets the starting point which the <a>Unicode Bidirectional Algorithm</a> uses  to calculate the directions of the embedded levels. For more details, see <a href="http://www.unicode.org/reports/tr9/">Unicode Standard Annex #9, Unicode Bidirectional Algorithm</a> [[UAX9]], especially definitions BD2–BD5.<br><span class="see-also">See also the definition in the <a href="https://www.unicode.org/glossary/#paragraph_direction">Unicode Standard</a>.</span></p>
+
+<p><dfn data-lt="pre-base" class="lint-ignore export">Pre-base vowel</dfn>. A pre-base (or prescript) vowel glyph is displayed before the consonant or orthographic syllable after which it is pronounced. If the vowel character is a combining mark, it is still typed and stored in pronunciation order, and the application will render it in the correct location. In some scripts, such as Thai, a pre-base vowel glyph is represented by a normal letter, which is typed and stored in the correct position relative to the base.</p>
+
+<p><dfn class="lint-ignore export">Precomposed</dfn>. A precomposed character is one that can also be broken down into separate code points representing its component parts (decomposition). Typically this will include base characters with diacritics, such as accented Latin characters, or Indic characters with nuktas. [=Normalization Form C=] (NFC) produces precomposed characters from many [=decomposed=] sequences.</p>
+
+<p><dfn class="lint-ignore export">Producer</dfn>. When talking about strings on the Web, the W3C Internationalization group refers to a producer as any process where natural language string data is created for later storage, processing, or interchange.</p>
```

Comment:
Do we want to use a formula for talking about terms we invented (vs. just using prose)?

We might consider not exporting some terms that are "local to i18n"?